### DISCUSSION TITLE
App 내에서 사용되는 SF Symbols 관리 방법에 대한 논의가 필요합니다!

### DISCUSSION THREAD

**Discussion by HanGyeongjun** [Team-ShortcutsZip/MacC-Team-HappyAnding#405] [General]
Sprint 8에서 제가 가져간 이슈인 #46(SF Symbol 크키 통일)에 대해서 더 논의가 필요할 것 같습니다!
앱 전반에서 사용되는 SF Symbols의 크기와 두께를 일관성 있게 조절해야 한다는 큰 틀만 논의되었고, 자세한 기능들이 논의되지 않은 것 같아 팀원분들의 추가적인 의견이 필요합니다.

1. 현재 사용되고 있는 SF Symbols의 크기와 두께를 일정하게 조절하기만 해라!
2. 거기에 더해 폰트+텍스트처럼 extension과 열거형으로 관리하는거까지 해라!
3. 설마 이 이슈에서 더 할 게 있나요?

현재는 1. 처럼 크기 및 두께를 확인하고 조절만 해 주면 디자인적인 문제는 모두 해결된다고 생각합니다. 그리고 2. 의 경우 사용하는 SF Symbols가 훨씬 많아지게 되고, 검색이 필요할 경우 enum으로 관리하는게 쪼끔 더 편할거같다..는 생각이 드네요..? 그래서 2번의 경우 지금 당장보다는 SF Symbols가 추가된 이후가 적용시키기 좋은 시점일 것 같긴 합니다!

**Comment by HanGyeongjun** [comment]
지금 확인해보니 단축어 셀에 들어가는 친구들은 이미 extension으로 존재하는군요! 이러면 2. 까지 이번에 다 처리해버리는게 깔끔할지도 모르겠네요

**Comment by JIWON1923** [comment]
이번 스프린트에서 아이콘 추가 작업이 들어가니까 2번은 다음 스프린트에서 작업하는게 좋지 않나요?
그린 작업이랑 많이 겹칠 것 같아요

**Comment by HanGyeongjun** [comment]
지금 확인해보니 저희 디자인 시스템에 App에서 사용하는 심볼들에 대한 규칙이 잘 정리되어 있었네요..! 참고해서 2번까지 작업해놓겠습니다. 추가적인 의견 있으면 코멘트로 알려주세요!

**Reply by HanGyeongjun** [reply]
Font+Extension내부에 함께 작업 예정입니당

**Reply by JIWON1923** [reply]
좋십니도~
